odoo/addons/openupgrade_records/lib/compare.py

```diff
@@ -264,28 +268,68 @@ def match(match_fields, report_fields, warn=False):
 
 def compare_xml_sets(old_records, new_records):
     reprs = collections.defaultdict(list)
+
+    # direct match
     match_fields = ['module', 'model', 'name']
     for column in copy.copy(old_records):
         found = search(column, new_records, match_fields)
         if found:
             old_records.remove(column)
             new_records.remove(found)
 
+    # other module, same full xmlid
+    match_fields = ['model', 'name']
+    moved_records = []
+    for column in copy.copy(old_records):
+        found = search(column, new_records, match_fields)
+        if found:
+            old_records.remove(column)
+            new_records.remove(found)
+            column['old'] = True
+            found['new'] = True
+            column['moved'] = found['module']
+            found['moved'] = module_map(column['module'])
+            moved_records.append(column)
+            moved_records.append(found)
+
+    # other module, same suffix, other prefix (with original_module == prefix)
+    match_fields = ['model', 'suffix', 'original_module']
+    renamed_records = []
+    for column in copy.copy(old_records):
+        found = search(column, new_records, match_fields)
+        if found:
+            old_records.remove(column)
+            new_records.remove(found)
+            column['old'] = True
+            found['new'] = True
+            column['renamed'] = found['module']
+            found['renamed'] = module_map(column['module'])
+            renamed_records.append(column)
+            renamed_records.append(found)
+
     for record in old_records:
         record['old'] = True
     for record in new_records:
         record['new'] = True
 
     sorted_records = sorted(
-        old_records + new_records,
+        old_records + new_records + moved_records + renamed_records,
         key=lambda k: (k['model'], 'old' in k, k['name'])
     )
     for entry in sorted_records:
         content = ''
         if 'old' in entry:
             content = 'DEL %(model)s: %(name)s' % entry
+            if 'moved' in entry:
+                content += ' [potentially moved to %(moved)s module]' % entry
+            elif 'renamed' in entry:
+                content += ' [renamed to %(renamed)s module]' % entry
```

Comment:
The word `potentially` should be also on this side

Reply:
the rename is when the prefix (module name) changed. It's difficult to have a wrong rename :S

Reply:
I don't agree. You can have the same suffix for different things.

Reply:
also when it's a rename you should use rename_xmlid method. This is not the case of moved xmlids

Reply:
We are mixing things here. Please answer first the main question outside this comment about the difference between renamed and moved.

Reply:
but it would be super improbable that in a module a xmlid is removed and then in another module (same model) the same xmlid appears but is used for a different thing

Reply:
well. I am lazy and don't want to discuss this. If you are happy with the "potentially renamed", then I will add it. No problem.

Reply:
Well, it was more a question of understanding the difference of both types. Now I understand, and as you have said, it's very improbable that, having that circumstances, isn't a rename, so you can leave it as before, or change the analysis file according the new text. What you prefer.

Reply:
ok, changed again haha